balances-tx-payment/src/balance_conversion.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2019 Alain Brenzikofer
+// This file is part of Encointer
+//
+// Encointer is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Encointer is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Encointer.  If not, see <http://www.gnu.org/licenses/>.
+
+use crate::{AssetBalanceOf, AssetIdOf, BalanceOf};
+use core::marker::PhantomData;
+use encointer_primitives::{balances::EncointerBalanceConverter, communities::CommunityIdentifier};
+use frame_support::traits::tokens::BalanceConversion;
+use pallet_encointer_balances::Pallet as BalancesPallet;
+use pallet_encointer_communities::{Config as CommunitiesConfig, Pallet as CommunitiesPallet};
+use sp_runtime::traits::Convert;
+
+/// Transforms the native token to the community currency
+///
+/// Assumptions:
+/// * Native token has 12 decimals
+/// * fee_conversion_factor is in Units 1 / [KSM]
+pub fn balance_to_community_balance(
+	balance: u128, // Unit = [pKSM]
+	reward: u128,
+	fee_conversion_factor: u32,
+) -> u128 {
```

Comment:
Please comment how a u128 will be interpreted as a CC balance U64F64
Like this it is difficultbto interpret the formula

Reply:
in this step there is actually no U64F64 involved. we convert between a `u128` KSM value and a `u128` fungible balance